### ISSUE TITLE
Remove CLDR from the submodules

### ISSUE THREAD

**Issue by OzancanKaratas** [nvaccess/nvda#10955]
### Steps to reproduce:
CLDR files should be updated manually.
### Actual behavior:
Submodule repository is not always updated.
### Expected behavior:
NVDA's CLDR repository should be always updated. We should not rely on third-parties.
CC: @feerrenrut, @josephsl

**Comment by josephsl** [comment]
Actually, CC @LeonardDer

**Comment by leonardder** [comment]
Including the CLDR data as a submodule is on purpose. As soon as the source repository is updated, updating our cldr data is as easy as updating the reference to the submodule. I don't see any reason to change this, other than the source repository we're using not being updated as soon as new CLDR data arrives, but in the past, that turned out to be as easy as poking the maintainer with a Github issue.

Therefore, I'm closing this. If you want CLDR data to be updated right now, that's a different subject, but I will look into it instantly.

**Comment by leonardder** [comment]
See https://github.com/fujiwarat/cldr-emoji-annotation/issues/3

**Comment by OzancanKaratas** [comment]
@leonardder, The submodule repository is not updated simultaneously. I think we should update it manually, if you let me, I can do that. So we don't have to trust someone else.

**Comment by leonardder** [comment]
We can do something like that if @fujiwarat is no longer able or maintain his repo, which I'm currently not aware of. However, in that case the way to go will probably be that the repository will be forked to the NVAccess organisation.
Let's rre-eevaluate this in a week or so.

**Comment by OzancanKaratas** [comment]
@leonardder, We should use the official CLDR repository. Please assign me. I want to work for job I mentioned. @fujiwarat said that it will not be maintained his own CLDR repository. See our discussion [here](https://github.com/fujiwarat/cldr-emoji-annotation/pull/4).

**Comment by leonardder** [comment]
I can't assign you officially, but feel free to go ahead! Probably best to close your current cldr pr, then?
